bundle/manifests/3scale-operator.clusterserviceversion.yaml

```diff
@@ -240,12 +240,13 @@ metadata:
     categories: Integration & Delivery
     certified: "false"
     containerImage: quay.io/3scale/3scale-operator:master
-    createdAt: "2019-05-30T22:40:00Z"
+    createdAt: "2024-11-27T13:51:25Z"
```

Comment:
as above, if we remove it from base we won't have it, I don't think this is a mandatory field for csv validation. Please remove